lib/helpers/isRootCatchAllRedirect.js

```diff
@@ -0,0 +1,5 @@
+// Return true if the redirect is a root level catch-all
+// (e.g., /[...slug] or [[...slug]])
```

Comment:
actually [...slug] wouldn't cause this issue, because it requires at least one parameter! Only optional catch-all route creates this "/*" redirect and issue.